mempool/reactor_test.go

```diff
@@ -117,7 +120,7 @@ func TestReactorConcurrency(t *testing.T) {
 		}()
 
 		// 1. flush the mempool
-		reactors[1].mempool.Flush()
+		// reactors[1].mempool.Flush()
```

Comment:
if I remember correctly, flushing the mempool was giving us test flake, that's why removed the flushing.

Reply:
i kept running into the same, realised it was uncommented in core so made the change to match